app/views/assets/_stream_bed_profile.html.haml

```diff
@@ -12,7 +12,7 @@
       .panel-heading
         %h3.panel-title
           %a{data: {toggle: 'collapse'}, href: '#collapse-streambed'}Streambed Profile
-          - if can?(:update, @asset) && can?(:manage, StreambedProfile) && !@asset.assigned_version
+          - if can?(:update_from_structure, @asset) && !@asset.assigned_version && @asset.streambed_profiles.map{|p| p.inspection&.state}.any?('final')
```

Comment:
Not worth changing now, but in the future, `@asset.streambed_profiles.map{|p| p.inspection&.state}.any?('final')` could be implemented as `@asset.streambed_profiles.any?{|p| p.inspection&.state == 'final'}`. That's a little more efficient because it will return as soon as the first profile with a final inspection is found, rather than instantiating all the inspections (to get the state values) and then seeing if 'final' is in that list.